client/src/components/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,48 @@
+import { ChangeEvent, useState } from 'react';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import DeleteIcon from '@material-ui/icons/Delete';
+import useStyles from './useStyles';
+import { User } from '../../interface/User';
+import ProfileAvatar from '../ProfileAvatar/ProfileAvatar';
+
+interface Props {
+  loggedInUser: User;
+}
+
+const ProfilePhoto = ({ loggedInUser }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Paper elevation={5} className={classes.root}>
+      <Grid container justify="center">
+        <Grid item container lg={4} md={6} xs={11} direction="column" alignItems="center">
+          <Box className={classes.title}>
```

Comment:
The `Typography` component is a block level component be default (I don't think we need these `Box` components here). Why don't we consider adding CSS classes to it directly?